go.mod

```diff
@@ -59,25 +59,88 @@ require (
 )
 
 require (
+	github.com/calmh/incontainer v0.0.0-20221224152218-b3e71b103d7a
+	github.com/duo-labs/webauthn v0.0.0-20221205164246-ebaf9b74c6ec
```

Comment:
This is an archived repo, with it's readme point out this as it's successor: https://github.com/go-webauthn/webauthn

Reply:
Yeah, I noticed but haven't gotten around to migrating it yet. I'll put it on the todo list.